Create 2025-04-23.md (#728)
solid/specification: meetings/2025-04-23.md

# W3C Solid Community Group: Weekly

* Date: 2025-04-23T14:00:00Z
* Call: https://meet.jit.si/solid-cg
* Chat: https://matrix.to/#/#solid_specification:gitter.im
* Repository: https://github.com/solid/specification

## Present
* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
* Alain Bourgeois
* Hadrian Zbarcea
* [Rahul Gupta](https://cxres.pages.dev/profile#i)
* Tom Byrd

> * Ted

Reviewer: ```suggestion
* [TallTed // Ted Thibodeau Jr](https://github.com/TallTed/) (he/him) ([OpenLink Software](https://www.openlinksw.com/))
```

* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Hadrian
* Pavlik

### Introductions
* Tom Byrd, IT professional for 40 years. Activity involved in privacy rights. Came across Solid in 2021. Working on a few ideas, want to be in sync with Solid and ODI.

---

## Topics

### Solid Symposium 2025

https://sosy2025.eu/

* eP: April 24-25
* eP: Will there be materials available online afterwards?
* ... I believe Jeff, Rahul and Jesse have sessions.
* RG: Solid Practitioners are organizing a session there. There are 6 talks. We try to foster collaboration between communities.
* ... Speakers from EU and US. 
* https://sosy2025.pages.dev/solid-together
* eP: Good luck.
* RG: It's tomorrow morning 10am CET.
* ... I intend to put up the slides. We probably won't have video. We'll try to broadcast, but we won't have the ability to share the link too much in advance.

### Code of Conduct related issues

> * eP: Tomorrow Hadrian and I will have a call with W3C staff. Michiel is on vacations.
> * eP: If you would like to bring up interactions where CoC might have been violated. You can do it publically or contact CG chairs directly. 
> * RG: Please refrain from discussing any issue that involves me until I am sent an official communication from W3C and the Solid CG.

Reviewer: ```suggestion
* eP: Tomorrow Hadrian and I will have a call with W3C staff. Michiel is on vacation.
* eP: If you would like to bring up interactions where CoC might have been violated. You can do it publicly or contact CG chairs directly. 
* RG: Please refrain from discussing any issue that involves me until I am sent an official communication from W3C and the Solid CG.
```

> * HZ: Based on my experience with Apache, we talk about issues not people in public. In general people mean well just emotions get escalated. I think we will resolve it and my goal is to prevent such problems from haunting us in the fugure. We are a community and we have to communicate.
> * HZ: In general I don't want to offer to mediate, unless I'm asked. This might be seen as pressure from one of the sides. Please ask me if you want help with resolving the problem.
> * RG: I haven't received any communication except the two public comments. I don't think the issues should be discussed until the situation is resolved. I have right to privacy and ask for a right to privacy. 

Reviewer: ```suggestion
* HZ: Based on my experience with Apache, we talk about issues not people in public. In general, people mean well; emotions just get escalated. I think we will resolve it and my goal is to prevent such problems from haunting us in the future. We are a community, and we have to communicate.
* ... In general, I don't want to offer to mediate, unless I'm asked. This might be seen as pressure from one of the sides. Please ask me if you want help with resolving the problem.
* RG: I haven't received any communication except the two public comments. I don't think the issues should be discussed until the situation is resolved. I have right to privacy, and ask for a right to privacy. 
```

> * eP: We are trying to update everybody, in the minutes and on matrix. It's not about a person in particular, it's about the whole situation.
> * HZ: I don't think we can resolve anything now. Rahul your point was understood.
> * eP: We'll know better how to move forward after tomorrow's meeting. I think it's important to have a good working environment.

Reviewer: ```suggestion
* eP: We are trying to update everybody, in the minutes and on matrix. It's not about one person in particular; it's about the whole situation.
* HZ: I don't think we can resolve anything now. Rahul, your point was understood.
* eP: We'll know better how to move forward after tomorrow's meeting. I think it's important to have a good working environment.
```

### Backup/Restore Issue history

https://github.com/solid/specification/issues?q=is%3Aissue%20state%3Aopen%20%22copyright%22%20in%3Atitle%20

https://github.com/solid/specification/issues/454#issuecomment-2666339495

> + RG: Some previous participants have been deleting their comments in a high-handed manner (I am not against somene making an edit or even deleting a comment because they changed their minds). But when it is done after many years, this behaviour is disruptive. Of specific concern is the deletion headings in theses issues. 
> + RG: Valuable loss of context for those who joined the CG later. Can we please restore those comments, unless there is some exceptional legal reason to let them remain deleted. 
> + RG: The person is not even a CG member when they deleted the issues.
> * ... One person removed comments that were written years ago.

Reviewer: ```suggestion
+ RG: Some previous participants have been deleting their comments in a high-handed manner. (I am not against someone making an edit or even deleting a comment because they changed their minds.) But when it is done after many years, this behaviour is disruptive. Of specific concern, is the deletion headings in these issues. 
+ ... Valuable loss of context for those who joined the CG later. Can we please restore those comments? Unless there is some exceptional legal reason to let them remain deleted. 
+ ... The person is not even a CG member when they deleted the issues.
* ... One person removed comments that were written years ago.
```

* eP: I'll bring this issue up tomorrow as well, to get clarity from W3C about data retention.
* RG: Just find out what the rules are, and if in such a situation we need to revoke repository access to people who engage in this kind of behaviour. What are the rules? Do github rules apply?
* eP: Maybe there is a bot that archives everything. Maybe we need to have a separate system.

> * RG: If you wrote something a few days ago and want to correct that, updating would be accepted behavior. [ ](https://github.com/martinthomson/i-d-template)
> * HZ: I remember deleting some of my comments few minutes after writing them. It is very hard to draw a line when it is acceptable and when not. I look at trusting each other and acting in good faith. Before taking corrective actions we should always understand motive. Just talking about process is not a correct approach.
> * RG: If someone is contributing today, they may need the context from the past. Deleting past conversations disrupts capacity to understand how things were arrived at.

Reviewer: ```suggestion
* RG: If you wrote something a few days ago and want to correct that, updating would be accepted behavior. https://github.com/martinthomson/i-d-template
* HZ: I remember deleting some of my comments few minutes after writing them. It is very hard to draw a line when it is acceptable and when not. I look at trusting each other and acting in good faith. Before taking corrective actions, we should always understand motive. Just talking about process is not a correct approach.
* RG: If someone is contributing today, they may need the context from the past. Deleting past conversations disrupts capacity to understand how things were arrived at.
```

> * HZ: If I remove my past comment? Does it impact ability to understand the spec? We need to be careful not to overblown the issue. 
> * eP: I agree that we should not just look at processes. At the same time, if we have a process we could use it as a guideline. I think there is a sweetspot in between. Let's try to find a way out in a graceful manner.

Reviewer: ```suggestion
* HZ: If I remove my past comment, does it impact others' ability to understand the spec? We need to be careful the issue is not overblown. 
* eP: I agree that we should not just look at processes. At the same time, if we have a process, we could use it as a guideline. I think there is a sweet spot inbetween. Let's try to find a way out in a graceful manner.
```

* RG: You need good public records for antecedents. No one is asking for sanctions on the person.

### QUERY to fetch Auxiliary resources

https://github.com/solid/specification/issues/724

* eP: posponed until CoC process moves forward

### SHQ Notifications

https://github.com/solid/notifications/pull/201

* eP: posponed until CoC process moves forward

### Curated list of known CSS extensions

https://github.com/CommunitySolidServer/CommunitySolidServer/discussions/2024#discussioncomment-12837567

* eP: Joachim responded
* ... his suggestion was a separate repo that could be easier to maintain and linked from the Solid repo.

> * AB: It looks nice, there is no objection. I will propose to Jesse to create in solid-contrib something like CSS extensions. I have the list, we can have direct access to these extensions somewhere. I know that for the google extensions there is no objection to transfer, maintenance will be required longer term.
> * eP: would AB like to be a maintainer for the repo.

Reviewer: ```suggestion
* AB: It looks nice, there is no objection. I will propose to Jesse that something like CSS extensions be created in `solid-contrib`. I have the list; we can have direct access to these extensions, somewhere. I know that for the google extensions, there is no objection to transfer. Maintenance will be required longer term.
* eP: would AB like to be a maintainer for the repo?
```
